docs/getting-started/csm-stack/image-to-ground-tutorial.ipynb

```diff
@@ -41,7 +41,7 @@
    "metadata": {},
    "source": [
     "```\n",
-    "conda install -c conda-forge knoten=0.2.1\n",
```

Comment:
we want to have limits on versions to make them more repeatable. Otherwise an update would break the tutorial

Reply:
0.4 pinned.